CHANGELOG.md

```diff
@@ -18,6 +18,10 @@
   [Paul Beusterien](https://github.com/paulb777)
   [Andrew](https://github.com/mad-rain)
   [#739](https://github.com/CocoaPods/Xcodeproj/pull/739)
+  
+* Fix crash when adding sub project.
```

Comment:
nit: add two empty spaces at the end of this line for markdown formatting.

Reply:
I added it, thank you for your reminder.